UI/users/preferences.html

```diff
@@ -80,17 +82,18 @@
                                   default_values = [ user.prefs.numberformat ],
                                   text_attr = 'format',
                                   value_attr = 'format',
+                                  required = 1
                                 } %]</td>
                           </tr>
                           <tr>
                             <th align="right"><label for="language">[% text('Language') %]</label></th>
-                            <td>[% PROCESS select_language element_data={
+                            <td>[% PROCESS select element_data={
                                 name = 'language',
                                 options = user.language_codes,
                                 default_values = [ user.prefs.language ],
                                 text_attr = 'label',
                                 value_attr = 'id',
-                                default_blank = 1,
```

Comment:
Why isn't `default_blank = 1` a good idea? As long as no language has been selected, this will result in the first item being selected. I think offering an empty selection, failing to submit, is better than silently saving the first choice.

Reply:
Agreed. Fixed